### DISCUSSION TITLE
Out of my cave, lots of updates · poolp.org

### DISCUSSION THREAD

**Discussion by giscus** [poolpOrg/poolp.org#171] [Articles]
# Out of my cave, lots of updates · poolp.org

TL;DR: I have been silent for a while due to personal matters but I did a ton of stuff on OpenSMTPD, plakar and a handful of other projects.

https://poolp.org/posts/2024-06-11/out-of-my-cave-lots-of-updates/

**Comment by simonhuk** [comment]
Good to see you back! Hope this year is better for you and your family.

**Reply by poolpOrg** [reply]
thanks :-)
